packages/devtools_app/lib/src/screens/memory/panes/profile/profile_view.dart

```diff
@@ -671,6 +671,8 @@ class _ProfileHelpLink extends StatelessWidget {
           const Text('The allocation profile tab displays information about\n'
               'allocated objects in the Dart heap of the selected\n'
               'isolate.'),
+          const SizedBox(height: denseSpacing),
+          const ClassTypeLegend(),
           MoreInfoLink(
             url: DocLinks.profile.value,
             gaScreenName: '',
```

Comment:
This _ProfileHelpLink widget is duplicated below. We can create a shared widget MemoryTabHelp to share this code across tabs.

Reply:
The structure is similar, but the is different. It can become more different in future. And it is just for two tabs. I prefer not to generalize for now.